src/profilers/profile-serializer.ts

```diff
@@ -15,132 +15,278 @@
  */
 
 import {perftools} from '../profile';
-import {getIndexOrAdd} from '../util';
-import {TimeProfile, TimeProfileNode} from '../v8-types';
-// A stack of function UIDs.
+import {AllocationProfileNode, ProfileNode, TimeProfile, TimeProfileNode} from '../v8-types';
+
+// A stack of function IDs.
 type Stack = Array<number>;
+
+// A function which appends the sample form of entry to samples.
```

Comment:
Can't quite parse "appends the sample form of entry to samples", what does that mean? Could you rephrase?